### PR TITLE
QA markup/js fixes (IA-27 epic)

### PR DESCRIPTION
**Pull request by otecMuxah**
- random horisontal scrolls
- correct text
- fixed gradebook table not equal cells width
- refactored Cohort js

ia-(61-65)

### CODE REVIEW

File: rg_instructor_analytics/static/rg_instructor_analytics/js/CohortTab.js [hunk 1]
```diff
@@ -34,26 +34,24 @@ function CohortTab(button, content) {
...
-                    '<input name="cohort-checkbox" type="checkbox" value="' +
-                    response.cohorts[i].students_id + '">' +
-                    '<label style="display: inline-block;">' +
-                    '&ensp;' + response.labels[i] + ':&ensp;' + response.cohorts[i].students_username +
-                    '</label>' +
-                    '</div>' +
-                    '</li>'
+                    `<li>
+                        <div>
+                            <input id="${item}" name="cohort-checkbox" type="checkbox" value="${response.cohorts[i].students_id}">
```
Comment (idegtiarov) [inline rg_instructor_analytics/static/rg_instructor_analytics/js/CohortTab.js]: Too long lines please make them shorter.

File: rg_instructor_analytics/static/rg_instructor_analytics/js/GradebookTab.js [hunk 2]
```diff
@@ -74,6 +74,20 @@ function GradebookTab(button, content) {
 
         greadebookTab.gradebookTableBody.append(htmlStringStudents);
 
+        //Make cells width equal to biggest cell
+        let maxLength = 0;
+        let $tableCells = $('.gradebook-table-cell:not(:first-child)');
+
+        $tableCells.each((item)=>{
```
Comment (idegtiarov) [inline rg_instructor_analytics/static/rg_instructor_analytics/js/GradebookTab.js]: I would add spaces around `=>`

File: rg_instructor_analytics/static/rg_instructor_analytics/js/GradebookTab.js [hunk 3]
```diff
@@ -74,6 +74,20 @@ function GradebookTab(button, content) {
...
+            let width = $tableCells[item].clientWidth;
+            if (maxLength < width) {
+                maxLength = width;
+            }
+        });
+
+        $tableCells.each((item)=>{
```
Comment (idegtiarov) [inline rg_instructor_analytics/static/rg_instructor_analytics/js/GradebookTab.js]: ditto

**Review by idegtiarov — CHANGES_REQUESTED** [review]

File: rg_instructor_analytics/static/rg_instructor_analytics/js/CohortTab.js (hunk 1 above)
Comment (otecMuxah) [reply rg_instructor_analytics/static/rg_instructor_analytics/js/CohortTab.js]: done

File: rg_instructor_analytics/static/rg_instructor_analytics/js/GradebookTab.js (hunk 2 above)
Comment (otecMuxah) [reply rg_instructor_analytics/static/rg_instructor_analytics/js/GradebookTab.js]: done

File: rg_instructor_analytics/static/rg_instructor_analytics/js/GradebookTab.js (hunk 3 above)
Comment (otecMuxah) [reply rg_instructor_analytics/static/rg_instructor_analytics/js/GradebookTab.js]: done

**Review by idegtiarov — APPROVED** [review]

**Pull request merged**